.travis.yml

```diff
@@ -7,8 +7,11 @@ matrix:
     install:
       - pip install -r requirements.txt
       - pip install python-coveralls
+    env: COVERALLS_PARALLEL=true
     script:
       - pytest --cov=mitxgraders --cov-report=term-missing
+    notifications:
```

Comment:
Try removing the notification and webhook lines. We're using https://github.com/z4r/python-coveralls/tree/master to run coveralls, and it has parallel built in (see readme file at that repo).

Reply:
I've been looking at that and related threads for a while 🙍‍♂ 

This seems promising but more complicated: https://github.com/travis-ci/beta-features/issues/42

Trouble with just using the `combine` command from `coverage` is that it needs files written by `pytest`, which aren't shared between Travis stages, so they need to be uploaded somewhere (AWS S3 in the linked issue.)